modular_skyrat/modules/events/code/modules/events/meteor_wave.dm

```diff
@@ -0,0 +1,3 @@
+/datum/round_event/meteor_wave/setup()
+	startWhen = rand(90, 180) // Apparently it is by 2 seconds, so 90 is actually 180 seconds, and 180 is 360 seconds. So this is 3-6 minutes
```

Comment:
180 seconds is not enough to order sats and get them up in time at all. Currently even 220 seconds often doesn't get it up in time when cargo isn't perfectly on the ball.

Consider upping the lower limit to 4-4.5 minutes at least.